docs/compute/decentralized_live_migration.md

```diff
@@ -0,0 +1,248 @@
+3. **Expediting Restores:** Instant restore capabilities from backup vendors, particularly when coupled with namespace migration, can drastically speed up recovery times. VMs can be quickly restored to a temporary location and then migrated to their original namespace and storage.
+
+This document assumes the presence of an orchestrator for managing resource creation and removal. While future KubeVirt versions will incorporate an orchestrator for decentralized live migrations, the examples provided here detail the underlying steps an orchestrator would perform rather than utilizing one directly.
+
+### Decentralized live migration Feature Gate
+
+Decentralized live migration support must be enabled in the feature gates to be available. The [feature gates](../cluster_admin/activating_feature_gates.md#how-to-activate-a-feature-gate)
+field in the KubeVirt CR must be expanded by adding the `DecentralizedLiveMigration` to it.
+
+# Architecture
+
+**Compute live migrations**
+
+Compute live migrations are characterized by a single `VirtualMachineInstance` resource and a single `VirtualMachineInstanceMigrations` resource. A migration is initiated by creating a `VirtualMachineInstanceMigration` resource, with coordination occurring via the VirtualMachineInstance's status, as shown in Figure 1.
+
+Figure 1:
+![Figure 1](live-migration.png)
+
+**Storage live migrations**
+
+Storage live migrations, a variation of compute live migrations, involve copying the virtual machine's state and disk contents over the migration network. More information on initiating a storage live migration can be found in this [document](https://kubevirt.io/user-guide/storage/volume_migration/).
+
+**Decentralized live migrations**
+
+Decentralized live migrations are a variation of storage live migrations. They involve two `VirtualMachineInstances` and two `VirtualMachineInstanceMigrations`. Similar to storage live migrations, disk contents are copied over the network to the receiving virtual machine. However, the key difference is that the receiving virtual machine has a completely separate `VirtualMachineInstance`. In order to coordinate the migration, the status of the source and target 'VirtualMachineInstances` have to be synchronized, as illustrated in Figure 2.
+
+Figure 2:
+![Figure 2](decentralized-live-migration.png)
+
+A dedicated synchronization controller facilitates communication between the source and target `VirtualMachineInstances`, as depicted in Figure 2. This design enables the reuse of most existing live migration mechanisms developed for `compute` migrations.
+
+# Required resources
+
+To migrate a virtual machine (VM) between namespaces or clusters, you must create both a target `VirtualMachine` resource and a target `VirtualMachineInstanceMigration` resource. A decentralized live migration cannot be achieved using only a `VirtualMachineInstance` resource.
```

Comment:
"To migrate a virtual machine (VM)"
I appreciate this, however it would be better off at the top for first use. 
Funnily enough, you've used "virtual machine" for nearly all instances throughout this file. I'm largely ambivalent, since there's a case for both (and 'VM' outweighs 'virtual machine' quite heavily across the docs as a whole) but we may as well be consistent in each markdown file. For instance, L6 sneaks in a 'VM' but then reverts back to 'virtual machine'. 
I'd call this a nit :)